### DISCUSSION TITLE
What is the location for custom language files?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#224] [General]
I'm using ecode in portable mode with `portable_mode.txt` and the document doesn't cover it:

https://github.com/SpartanJ/ecode?tab=readme-ov-file#custom-languages-support

`config/terminal` has no `languages` directory. But there is a `languages` directory in `config`.

**Comment by ghost** [comment]
@SpartanJ

**Reply by SpartanJ** [reply]
It will located alongside the ecode binary, basically what usually is at `~/.config/ecode` it will go at `$ECODE_PATH/config`. Folder should be created after running in portable mode. So languages will be searched at `$ECODE_PATH/config/languages`.
